staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -251,6 +251,9 @@ func (o *ApplyOptions) Complete(f cmdutil.Factory, cmd *cobra.Command) error {
 
 	o.OpenAPISchema, _ = f.OpenAPISchema()
 	o.Validator, err = f.Validator(cmdutil.GetFlagBool(cmd, "validate"))
+	if err != nil {
```

Comment:
This one is fine. It returns an interface, which I assume will be nil if an error is returned, it would crash if it fails.